include/EASTL/sort.h

```diff
@@ -223,7 +223,7 @@ namespace eastl
 
 			for(++iSorted; iSorted != last; ++iSorted)
 			{
-				const value_type temp(*iSorted);
+				value_type temp(eastl::forward<value_type>(*iSorted));
```

Comment:
The problem isn't really obvious unless you look down here on line 238:
```
*iNext = eastl::forward<value_type>(temp);
```

If `temp` is a `const value_type` and `iNext` is only move-assignable then assigning a const value is incompatible.